samples/azure/azure-container-instances-dab-starter.ps1

```diff
@@ -0,0 +1,705 @@
+    docker push "${acrLoginServer}/${acrImageName}:${acrImageTag}"
+    Write-Success "Image pushed to ACR"
+
+    # Create SQL Server
+    Write-Step "Creating SQL Server..."
+    az sql server create `
+        --name $sqlServerName `
+        --resource-group $ResourceGroup `
+        --location $Location `
+        --admin-user $sqlAdminUser `
+        --admin-password $sqlAdminPasswordPlain | Out-Null
+    Write-Success "SQL Server created: $sqlServerName"
+
+    # Create SQL Database
+    Write-Step "Creating SQL Database..."
+    az sql db create `
+        --resource-group $ResourceGroup `
+        --server $sqlServerName `
+        --name $sqlDbName `
+        --service-objective $SqlServiceTier | Out-Null
+    Write-Success "SQL Database created: $sqlDbName"
+
+    # Configure firewall rules
+    Write-Step "Configuring SQL Server firewall rules..."
+    
+    # Allow Azure services (correct range for Azure services)
+    az sql server firewall-rule create `
+        --resource-group $ResourceGroup `
+        --server $sqlServerName `
+        --name "AllowAzureServices" `
+        --start-ip-address 0.0.0.0 `
+        --end-ip-address 0.0.0.0 | Out-Null
+    
+    # Allow client IP
+    $myIp = (Invoke-WebRequest -Uri "https://api.ipify.org" -UseBasicParsing).Content.Trim()
+    az sql server firewall-rule create `
+        --resource-group $ResourceGroup `
+        --server $sqlServerName `
+        --name "ClientIP" `
+        --start-ip-address $myIp `
+        --end-ip-address $myIp | Out-Null
+    
+    Write-Success "Firewall rules configured (Azure services + your IP: $myIp)"
+
+    # Load test data
+    Write-Step "Loading test data into database..."
+    $sqlServer = "$sqlServerName.database.windows.net"
+    
+    # Wait for firewall rules to propagate (Azure can take up to 5 minutes)
+    Write-Host "Waiting for firewall rules to propagate..." -ForegroundColor Yellow
+    Start-Sleep -Seconds 30
+    
+    $maxRetries = 3
+    $retryCount = 0
+    $dataLoaded = $false
+    
+    while (-not $dataLoaded -and $retryCount -lt $maxRetries) {
+        try {
+            if ($retryCount -gt 0) {
+                Write-Host "Retry attempt $retryCount of $maxRetries..." -ForegroundColor Yellow
+            }
+            
+$output = sqlcmd -S $sqlServer -U $sqlAdminUser -P $sqlAdminPasswordPlain -d $sqlDbName -i $sqlScriptPath -I -b 2>&1
+            
+            if ($LASTEXITCODE -eq 0) {
+                $dataLoaded = $true
+                Write-Success "Test data loaded successfully"
+            }
+            else {
+                throw "sqlcmd returned exit code $LASTEXITCODE"
+            }
+        }
+        catch {
+            $errorMessage = $_.Exception.Message
+            if ($output) { $errorMessage = $output | Out-String }
+            
+            # Check if error is due to IP not allowed
+            if ($errorMessage -match "Client with IP address '([0-9.]+)' is not allowed") {
```

Comment:
this case is not handled in the ACA script. I think we need to move the common steps into a single script, and the 2 scripts contain only the ACI/ACA specific deployment steps.